docs/ide-setup.md

```diff
@@ -108,22 +108,22 @@ You can sign up for the free [JetBrains student license](https://www.jetbrains.c
 
    * Click `Configure → Project Defaults → Project Structure` (or `File → Project Structure` if a project is currently open).
      Select SDKs in Platform Settings and check if there is an SDK named `1.8` with a JDK home path pointing to a JDK 8 path.
-     Otherwise add a new SDK using JDK 8 with a name of `1.8`.
+     Otherwise, add a new SDK using JDK 8 with a name of `1.8`.
      ![intellijsetupguide-1.png](images/intellijsetupguide-1.png)
 
 1. You need the [Google Cloud Tools](https://cloud.google.com/tools/intellij/docs/quickstart-IDEA#install) plugin installed and configured:
 
    ![intellijsetupguide-2.png](images/intellijsetupguide-2.png)
    * During installation, you may encounter a prompt to disable the obsolete `Google App Engine Integration` plugin. Answer `Yes`.
    * After installation, restart IntelliJ IDEA and configure the plugin.
-     Click `Configure → Settings/Preferences` (or `IntelliJ IDEA → Preferences` if a project is currently open),
+     Click `Configure → Settings/Preferences` (or `IntelliJ IDEA → Preferences` if the project is open),
```

Comment:
The meaning has changed, originally it is supposed to mean if any project is open and not the specific project.

Reply:
does "if any project is open" sound better?

Reply:
I suppose that could work. I don't have any better ideas myself unless anyone has a better suggestion?

Reply:
changed them all to "if a project is open"